cmd/crc/cmd/start.go

```diff
@@ -55,3 +52,12 @@ func runStart(arguments []string) {
 		logging.ErrorF(err.Error())
 	}
 }
+
+func validateDriver(driver string) {
+	for _, d := range preflight.SupportedVMDrivers {
```

Comment:
So this would for now allow VirtualBox on Windows... and we are only targeting VirtualBox on macOS for now.